Sources/SwiftProtobuf/UnknownStorage.swift

```diff
@@ -17,7 +17,7 @@
 import Foundation
 
 public struct UnknownStorage: Equatable {
-  internal var data = Data()
+  public private(set) var data = Data()
```

Comment:
`internal(set)`? (Sorry, didn't realize it was originally internal, not private, when we chatted offline.)